Update PHP 7.2 to 7.3 in panel Getting Started guide

vilhelmprytz opened the pull request:
See https://github.com/pterodactyl/documentation/issues/158 for reference.

TrixterTheTux commented:
You'd also want to update the php-fpm socket path in the vhost config as otherwise people will get bad gateway.

VilhelmPrytz commented:
@TrixterTheTux Thanks for pointing out. I've added a new commit.

I replace 7.2 with 7.3 in the nginx configurations. A file named nginx-php7.3.conf already existed, so I deleted that one and updated all references for that file to point to the default file instead (I checked with diff before, it had not been modified in any other way).

Should be good now!

Cheers

VilhelmPrytz commented:
Any updates on this?
